### PR TITLE
Make DeprecatedFrameworkWindow, PackageManagmentWindow and NuGetProjectUpgradeWindow accessible

### PR DESCRIPTION
**Pull request by PatoBeltran**
Added support for WPF automation api in the UI for these three windows
and made sure they had a good keyboard navigation and where usable with
screen readers such as narrator and jaws

### CODE REVIEW

**Review by jainaashish — APPROVED** [review]
LGTM

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx [hunk 1]
```diff
@@ -823,4 +823,13 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Accessibility_ThumbName" xml:space="preserve">
     <value>Thumb</value>
   </data>
+  <data name="ImageCaption_HelpIcon" xml:space="preserve">
+    <value>Help icon</value>
+  </data>
+  <data name="Accessibility_PromoteAllToTopLevel" xml:space="preserve">
+    <value>Promote all to top level dependency</value>
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx]: dependencies

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx [hunk 2]
```diff
@@ -823,4 +823,13 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
...
+  </data>
+  <data name="Accessibility_PromoteToTopLevel" xml:space="preserve">
+    <value>Promote to top level dependency</value>
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx]: to a top level dependency

**Review by nkolev92 — APPROVED** [review]
A couple of error message corrections.

**Pull request merged**